src/styles/index.scss

```diff
@@ -1,3 +1,9 @@
-body {
-  margin: 0;
-}
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
```

Comment:
Ensure that the URL for the font import is valid and accessible. If it's a placeholder or example, replace it with the actual URL to import fonts correctly. Remember that importing fonts directly in SCSS might not be the best approach for performance and might be better placed in the HTML as a link tag.